controllers/volumereplication_controller.go

```diff
@@ -84,9 +85,13 @@ func (r *VolumeReplicationReconciler) Reconcile(ctx context.Context, req ctrl.Re
 		return reconcile.Result{}, err
 	}
 
+	instance.Status.LastStartTime = metav1.Now()
+	r.updateStatus(instance, replicationv1alpha1.ReplicationState(""), "")
```

Comment:
I didn't understand this `ReplicationState(" ")` message call.  is this a way for returning empty ReplicationState type?

Reply:
yes

Reply:
Yes why this is set?

Reply:
Also, what happens when the resource is being deleted?  Do we still want to perform this particular update?

Reply:
Removed